proprietary_plugins/02_pro/smart_upload.md

```diff
@@ -23,20 +23,88 @@ To begin uploading, select _Smart Upload_ in the top right.
+
+#### What happens if multiple matches are found?
+
+Multiple endpoint to product matches is the primary purpose of smart upload! In the diagram above, a finding with multiple hosts/IPs will be split into identical findings and assigned to their respective products. If there are any endpoint hosts without a matching product, then they will also be split into separate findings by host.
+
+#### How is the data organized in a given product?
+
+Engagements and Tests created under a product are intended to be CI/CD focused. In most cases, the engagement will be reused. However, supplying updated arguments to the importer such as version, build ID, commit hash, or branch tag will be used to filter for a specific engagement based on those values. 
+
+![Engagements and Tests](../../../images/smart_upload/su-test-engagements.png)
+
+### Product Placement
+
+Findings parsed from a given report that are not matched to an existing product will be placed in a "product placement" queue where each finding can be assessed by there title, severity, endpoints where the vulnerability was detected, and what file the finding came from.
+
+![Product Placement](../../../images/smart_upload/su-placement.png)
+
+After selecting at least one finding, three options will be displayed:
 
 - _Create Product_ - create a new product for the finding to be assigned to
 - _Assign Product_ - list all existing products that the finding may be assigned to
-- _Disregard Findings_ - delete the unassigned finding altogether
+- _Discarded Findings_ - delete the unassigned finding altogether
 
-![Product Placement](../../../images/smart_upload/su-placement.png)
+#### Create Product
+
+Sometimes, a finding is detected on an endpoint that may not fully align with any existing products, or may not even fit product model directly. In these cases, it is most appropriate to create a new product for these findings to avoid having them getting misplaced in the future. After clicking the "Create Product" button, a short form will appear to gather some required information about the product:
+
+![Create Product](../../../images/smart_upload/su-create.png)
+
+More information can be added to this newly created product from the "Edit Product" page
+
+#### Assign Product
+
+In most cases with infrastructure scans, the endpoints where findings are discovered are often dynamic. When the location of a finding changes, it is necessary to create this link from this new location to an existing product so that a match can be found in the next import. After clicking the "Assign Product" button, a short form will appear to select the product this finding should be assigned to:
+
+![Assign Product](../../../images/smart_upload/su-assign.png)
 
-![Manually Assign Finding](../../../images/smart_upload/su-assign.png)
+#### Discarded Findings
 
-## API
+When a finding is deemed to be not worth keeping, then it can be discarded from the product placement queue altogether by click ing the "Discard Findings" button
```

Comment:
```suggestion
When a finding is deemed to be not worth keeping, then it can be discarded from the product placement queue altogether by clicking the "Discard Findings" button
```